editor/src/components/inspector/sections/event-handlers-section/event-handlers-section.tsx

```diff
@@ -57,7 +57,7 @@ export const EventHandlersSection = betterReactMemo('EventHandlersSection', () =
   }
 
   return (
-    <>
+    <React.Fragment>
```

Comment:
`<>` is actually a shorthand version to write `<React.Fragment>`

in some places in the codebase we use `<React.Fragment>` if we also use Emotion's css pragma (how to see if the pragma is used: the first line of a file will contain `/** @jsx jsx */` ). In other files, it is not necessary to write React.Fragment, you can use the `<></>` shorthand too.